### PR TITLE
docs: show simple example of migration from wapi-python

### PR DESCRIPTION
**Pull request by stigalvestad**
To show the most typical changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,10 +58,77 @@ make the switch.
...
+    data: TS = instance_curve.get_latest(data_to="2030-04-01T04:00Z")
+    print(f"Curve type: {data.curve_type}")
+    print(f"Time zone class of curve data: {type(data.tz)}")
+
+# Output
+# Curve type: INSTANCES
+# Time zone class of curve data: <class 'pytz.tzfile.CET'>
+```
+
+The migrated script is below
```
Comment (artchyn) [inline README.md:96]: Please, add a blank line here.

**Review by artchyn — APPROVED** [review]
Good update.

File: README.md [hunk 2]
```diff
@@ -58,10 +58,78 @@ make the switch.
...
+
+```python
+# python 3.9.8
+# pip install volue-insight-timeseries==1.2.0
+# pip install python-dotenv==1.0.1
```
Comment (deep-volue) [inline README.md:101]: Why do we need dotenv to demonstrate the example?

File: README.md [hunk 3]
```diff
@@ -58,10 +58,78 @@ make the switch.
...
+# CLIENT_SECRET=your-client-secret
+
+import os
+import volue_insight_timeseries as vit
+from volue_insight_timeseries.util import TS
+from volue_insight_timeseries.curves import InstanceCurve
+from dotenv import load_dotenv
+
+load_dotenv()
+session = vit.Session(client_id=(os.getenv('CLIENT_ID')),
```
Comment (deep-volue) [inline README.md:115]: I think the function `.getenv()` will be deprecated soon. Should rather use the dictionary directly `os.environ[]`

File: README.md [hunk 4]
```diff
@@ -58,10 +58,78 @@ make the switch.
...
+                      client_secret=(os.getenv('CLIENT_SECRET')))
+
+results = session.search(name="tt no2 con ec00 °c cet min15 f")
```
Comment (deep-volue) [inline README.md:118]: Would it be a good idea to check status first before fetching body?

**Review by deep-volue — COMMENTED** [review]
I know this is just an example. But I still did a code review like it is gonna be a production code. 🙂 Would be good to set a good example for clients.

File: README.md (hunk 4 above)
Comment (stigalvestad) [reply README.md:118]: This is not to show how we recommend users to write their code, but more about showing what is changing. I can add a comment about that in the top though.

File: README.md (hunk 2 above)
Comment (stigalvestad) [reply README.md:101]: It is needed to load environment variables from the .env file which is expected to be in the same folder. Makes the example simple to setup.

File: README.md (hunk 3 above)
Comment (stigalvestad) [reply README.md:115]: I agree it is better with os.environ, since I do not have a fallback value. Will update

**Comment by stigalvestad** [comment]
> I know this is just an example. But I still did a code review like it is gonna be a production code. 🙂 Would be good to set a good example for clients.

Making a good example of how to write a prod script is beyond the scope of this example.

**Pull request merged**